src/components/ChangePlan/Card/CardAction.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const CardAction = ({ url, children }) => {
+  return (
+    <a href={url} className="dp-button button-medium primary-green">
```

Comment:
Creo que en general, para este tipo de componentes siempre es buena idea permitir arrastrar propiedades arbitrarias, algo así:

```suggestion
const CardAction = ({ url, children, ...rest }) => {
  return (
    <a href={url} className="dp-button button-medium primary-green" {...rest}>
```

Aunque habría que ver que hacer con las clases, ya que creo que esto no funcionaría:

```html
<CardAction url={cardData.action.url} className="my-custom-class">{cardData.action.text}</CardAction>
```

Investigaste cual es la forma correcta o standard de hacer este tipo de cosas?

Reply:
por ahora le puse solo el rest, las clases se podrian manejar en el caso con la libreria `classnames` que agregue